pkg/registry/core/service/storage/storage.go

```diff
@@ -99,3 +104,8 @@ func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.Updat
 	// subresources should never allow create on update.
 	return r.store.Update(ctx, name, objInfo, createValidation, updateValidation, false, options)
 }
+
+// SetAfterDeleteFunc sets the given after delete func.
```

Comment:
Can these comments please explain why one sets r.AfterDelete and the other sets r.store.AfterDelete ?  This is very confusing to someone who isn't super familiar with the trickiness of Service